sphinx_needs/config.py

```diff
@@ -252,6 +252,7 @@ def __setattr__(self, name: str, value: Any) -> None:
     # add config for needs_id_builder
     build_json_per_id: bool = field(default=False, metadata={"rebuild": "html", "types": (bool,)})
     build_json_per_id_path: str = field(default="needs_id", metadata={"rebuild": "html", "types": (str,)})
+    lut_build_json: bool = field(default=False, metadata={"rebuild": "html", "types": (bool,)})
```

Comment:
```suggestion
    build_json_lut: bool = field(default=False, metadata={"rebuild": "html", "types": (bool,)})
```

This would be consistent with `build_json` and `build_json_per_id`